kmm/geodetic.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+from sweref99 import projections
+
+
+tm = projections.make_transverse_mercator("SWEREF_99_TM")
+
+
+def geodetic(positions: pd.DataFrame):
+    yx = positions.apply(
+        lambda row: (tm.grid_to_geodetic(row["x_sweref"], row["y_sweref"])),
```

Comment:
Shouldn't we call them longitude and latitude too so that we are consistent with the naming when adjusting meter/camera_meter?

Reply:
Ah sure, I didnt know they were also longitude/latitude :) Ill call them `{longitude/latitude}_{sweref/geodetic}` then.

These values do not affect adjustment of meter though?

Reply:
> Shouldn't we call them longitude and latitude too so that we are consistent with the naming when adjusting meter/camera_meter?

actually which are you referring to though, `x/y_sweref` or `yx`?

Reply:
Would prefer to call them just longitude/latitude to be consistent with how we dealt with adjusted meter

Reply:
My understanding though is that the SWEREF99 system is cartesian coordinates though and not angles like longitude/latitude. It would be kind of strange to call them longitude/latitude then